### PR TITLE
Sort imports and add docstrings

### LINKED ISSUE #171: doc strings are missing in lot of modules.
While checking with pylint, it gives a lot of suggestions to add doc strings in the functions.  
```************* Module tests.test_common
tests/test_common.py:1:0: C0111: Missing module docstring (missing-docstring)
tests/test_common.py:6:0: E0401: Unable to import 'PIL' (import-error)
tests/test_common.py:19:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:24:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:29:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:34:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:39:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:44:0: C0111: Missing function docstring (missing-docstring)
tests/test_common.py:49:0: C0111: Missing function docstring (missing-docstring)```

### PR DESCRIPTION
**Pull request by mzfr**
Fixes #171 
I have added docstrings and have sorted the imports in most of the file.

After this most of the error will be related to the snake_case format of argument or variables.

### CODE REVIEW

File: kodi_addon_checker/check_files.py [hunk 1]
```diff
@@ -6,14 +6,15 @@
     See LICENSES/README.md for more information.
 """
 
+import json
 import os
 import re
-import json
 import xml.etree.ElementTree as ET
 from pathlib import Path
```
Comment (LordGameleo) [inline kodi_addon_checker/check_files.py]: Please remove this line as ```Path``` is not used anywhere in this file. (unused import)

File: kodi_addon_checker/__main__.py [hunk 2]
```diff
@@ -7,15 +7,15 @@
...
+
+from kodi_addon_checker import check_addon, logger
 from kodi_addon_checker.check_repo import check_repo
 from kodi_addon_checker.common import load_plugins
-from kodi_addon_checker.config import ConfigManager, Config
-from kodi_addon_checker.record import Record, PROBLEM, WARNING, INFORMATION
+from kodi_addon_checker.config import Config, ConfigManager
+from kodi_addon_checker.record import INFORMATION, PROBLEM, WARNING, Record
 from kodi_addon_checker.report import Report
 from kodi_addon_checker.reporter import ReportManager
```
Comment (LordGameleo) [inline kodi_addon_checker/__main__.py]: Please remove ReportManager (unused-import)

File: kodi_addon_checker/schema_validation.py [hunk 3]
```diff
@@ -6,13 +6,14 @@
     See LICENSES/README.md for more information.
 """
 
-from os.path import abspath, dirname, join, exists, split
```
Comment (LordGameleo) [inline kodi_addon_checker/schema_validation.py]: Please remove abspath and split (unused-import)

File: kodi_addon_checker/check_py3_compatibility.py [hunk 4]
```diff
@@ -6,15 +6,13 @@
     See LICENSES/README.md for more information.
 """
 
-import os
-
-from lib2to3 import pgen2
-from lib2to3 import refactor
 import difflib
+import os
```
Comment (LordGameleo) [inline kodi_addon_checker/check_py3_compatibility.py]: Please remove os (unused-import)

**Comment by Razzeee** [comment]
@LordGameleo lets do that in another PR

@mzfr thank you!

**Comment by LordGameleo** [comment]
@Razzeee okay 😁

**Pull request merged**